### DISCUSSION TITLE
Percy Storybook config not recognised after updating to Storybook 6.5

### DISCUSSION THREAD

**Discussion by JodiWarren** [percy/percy-storybook#591] [Support] [answered]
## The problem

After updating to Storybook 6.5, the Percy CLI seems to no longer pick up the extended Percy config properly.

Running `npx percy config:validate` returns 

```
[percy] Found config file: .percy.yml
[percy] Invalid config:
[percy] - storybook: unknown property
```

`npx percy storybook ./docs --dry-run` results in around 1200 snapshots. We have numerous Excludes in our `percy.yml` file, which should bring the number down to around 600 results, which is what we getting before updating Storybook to 6.5.x

I've isolated this regression to have occured when we updated from Storybook 6.4.14 to 6.5.9. There were a few other Storybook-specific changes made as part of this update, but reverting them does not seem to affect the problem.

## Environment

- Node version: 17.5.0
- `@percy/cli` version: 1.3.1
- Version of Percy SDK you’re using: @percy/storybook: 4.2.1
- OS version: MacOS 12.4
- Type of shell command-line [interface]: zsh

**Comment by Robdel12** [comment, marked as answer]
👋🏼 Hey @JodiWarren! This typically happens when the `@percy/*` packages resolve to slightly different versions (due to weird dependency resolution from npm or yarn). I don't see this warning locally in our example app so I'm guessing that's what's happening here. You can delete your `node_modules` entirely and see if that fixes it after reinstalling. If not, you can install the SDKs fresh from the command line which should bring the lockfile inline again: `npm/yarn -D @percy/cli@latest @percy/storybook@latest`

**Reply by JodiWarren** [reply]
That's solved it for me, thank you! I just uninstalled and reinstalled both dependencies and voila! It works. Thank you again!
